molecule/default/converge.yml

```diff
@@ -2,6 +2,10 @@
 - name: Converge
   hosts: all
   roles:
+    - role: onaio.monit
+      slack_monit_endpoint: ""
+      monit_setup_mode: true
+      monit_scripts: ["monit"]
```

Comment:
Shouldn't this be postgres?

Reply:
Nope. The Monit role is called twice, first time here (where monit_setup_mode is set to true) and second time since it's a dependency in meta/main.yml (where monit_setup_mode is set to false). First call makes sure the service is installed and the second call installs the PostgreSQL specific Monit configurations.